src/components/todoList/index.js

```diff
@@ -0,0 +1,38 @@
+import React, { Fragment } from "react"
+import { useSelector } from "react-redux"
+import TodoItem from "components/todoItem"
+import AddTodoForm from "components/addTodoForm"
+import { IMAGE_PATHS, ADD_FIRST_TASK } from "utils/constants"
+import "components/todoList/todoList.scss"
+
+
+const TodoList = () => {
+  const selectTodos = (state) => state.todos.todoItems
+  const todos = useSelector(selectTodos)
+  const selectIsAdding = (state) => state.todos.isAdding
+  const toggleIsAdding = useSelector(selectIsAdding)
+
+  const renderedListItems = todos
+    .slice(0)
+    .reverse()
```

Comment:
Is there any reason behind reversing task array before mapping it into the DOM ?

Reply:
The tasks are being displayed in LIFO order. Meaning Last task added is being displayed as the first. This is why the array is being reversed bhaia.